cypress-tests/cypress/e2e/configs/Payout/Nomupay.js

```diff
@@ -0,0 +1,207 @@
+function get_billing(x) {
+  return {
+    address: {
+      line1: "1467",
+      line2: "Harrison Street",
+      line3: "Harrison Street",
+      city: "Munich",
+      state: "CA",
+      zip: "80331",
+      country: "DE",
+      first_name: "John",
+      last_name: "Doe",
+    },
+    phone: {
+      number: "9123456789",
+      country_code: "+91",
+    },
+    email: `payout_customer${Date.now() + x}@example.com`,
+  };
+}
+
+const card_data = {
```

Comment:
i would prefer maintaining `camelCase` everywhere in cypress going forward.

cc: @likhinbopanna